.github/workflows/lint.yml

```diff
@@ -0,0 +1,31 @@
+name: Lint
```

Comment:
Let's give this workflow a different name. How about `Spell Check`? "Lint" seems to be confusing with our code style linter.